src/app/views/forgotten-password/forgottenPasswordController.jsx

```diff
@@ -104,7 +110,7 @@ export class ForgottenPasswordController extends Component {
     requestEmailChange(email) {
         const body = { email: email };
         this.postResetPassword(body)
-            .then(response => {
```

Comment:
... just loud thinking, don't you want to check the response here?

Reply:
So kind of annoyingly the `http.post` which the `postResetPassword` calls uses another wrapper `./http-methods/request` which wraps the fetch API.

This request wrapper will check the response, if invalid it will attempt a number of retries. Then if they all fail and it gets anything outside the 200 rage it will throw an error, which will be picked up by the catch block under this.